internal/plugins/ansible/v1/plugin.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ansible
+
+import (
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+
+	"github.com/operator-framework/operator-sdk/internal/plugins"
+)
+
+const pluginName = "ansible" + plugins.DefaultNameQualifier
+
+var (
+	supportedProjectVersions = []string{config.Version3Alpha}
+	pluginVersion            = plugin.Version{Number: 1}
```

Comment:
```suggestion
	pluginVersion            = plugin.Version{Number: 1}
	pluginKey                = plugin.KeyFor(Plugin{})
```

Reply:
`pluginKey` wouldn't be used anywhere yet. I expect this to be added when help text is.